tests/widget/modal/widget-modal.login-2step.access.test.js

```diff
@@ -0,0 +1,43 @@
+import { expect, test } from '@playwright/test';
+
+test('2step login with access', async ({ page }) => {
```

Comment:
Here's the other instance of "access" that I think we should avoid. Either the full word "accessibility" or "a11y", I'd recommend.